composer.json

```diff
@@ -58,6 +58,7 @@
         "doctrine/orm": "^2.5",
         "lchrusciel/api-test-case": "^5.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.2.1",
+        "pagerfanta/pagerfanta": "^3.0",
```

Comment:
for now I suggest you to move it on main requirements. We'll handle the "optional" requirements later on next release

Reply:
it was just a revert to make our branches green again, so I would go with it for now. But I agree, that 1.9 needs this. The question is should be part of this PR?

Reply:
Hum good idea to make this change in another "Fix" PR.